viewing_party/party.py

```diff
@@ -1,23 +1,226 @@
 # ------------- WAVE 1 --------------------
+import json
+from unittest import result
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    Input: movie title, genre, and rating
+    Ouput: If values truthy, produces a dictionary of movies with title, genre, and rating key-value pairs.
+    If inputs are falsy, function returns None.
+    '''
+    movie_dict = {}
+    # if title == True and genre == True and rating == True:
+    if True: 
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+    if title == None or genre == None or rating == None:
+        return None
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+    '''
+    Input: user_data is a dictionary with a key "watched" and value of list of movie dictionaries the user has watched
+    Output: Function adds movie dictionary to user_data and returns user_data
+    '''
+    user_data["watched"] = [movie]
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    '''
+    Input: user_data is list of movie dictionaries with key "watchlist" and value of list of movie dictionaries
+    that the user wants to watch
+    Output: Function adds movie dictionary to "watchlist" and returns user_data
+    '''
+    user_data["watchlist"] = [movie]
+    return user_data
+
+def watch_movie(user_data, title):
+    '''
+    Input: user dictionary is a list of movie dictionaries with "watched" and "watchlist keys
+    representing the movies the user has watched or wants to watch. Title is a string representing
+    a movie the user has watched. 
+    Output: function moves movie represented by title to watched list if the user has that movie in the watchlist.
+    Function returns user_data.
+    '''
+    for movie_index in range(len(user_data["watchlist"])):

```

Comment:
Small note that should help with readability - doing `for i in range(len(iterable))` is great for when we want to use the index of each item. However, if we're only interested in the value(s) of each item, we can iterate over the values directly. 

Since you're just interested in the movie data and not the index of that movie, you could do something like this:
```
for movie in user_data["watchlist"]:
```
This also helps with readability. 😊